src/native/minipal/utils.h

```diff
@@ -50,13 +50,19 @@
 #    define DISABLE_ASAN
 #endif
 
+#if defined(_MSC_VER)
+#define CALLBACK_CALLCONV _cdecl
```

Comment:
I assume that this is only meant to be used for C-runtime callbacks like callbacks passed into bsearch or qsort method. It would not work as universal callback calling convention. There are number of callbacks on Windows that must be stdcall - there is `CALLBACK` define in Windows SDK that is defined as stdcall. 

Can we the name more specific? For example, `LIBC_CALLBACK_CALLCONV` or just `LIBC_CALLBACK`.